scripts/download-latest-graph.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+graph_path="otp_data/Graph.obj"
+
+if [[ ! -f "$graph_path" ]] || [[ "$*" == *"force"* ]]; then
+    echo "Pulling latest Graph.obj from S3..."
+    aws s3 cp "s3://cleanair-otp-data/Graph.obj" "$graph_path"
```

Comment:
```suggestion
    aws --profile=gophillygo s3 cp "s3://cleanair-otp-data/Graph.obj" "$graph_path"
```

Reply:
Thank you! That fix is added in [75ba6bb](https://github.com/azavea/cac-tripplanner/pull/1403/commits/75ba6bb3fe757afdfdd5ac189a74970f8e26cc46)